assets/xml/objects/object_gi_magicmushroom.xml

```diff
@@ -1,9 +1,14 @@
 ﻿<Root>
-    <File Name="object_gi_magicmushroom" Segment="6">
-        <DList Name="object_gi_magicmushroom_DL_000640" Offset="0x640" />
-        <DList Name="object_gi_magicmushroom_DL_000958" Offset="0x958" />
-        <Texture Name="object_gi_magicmushroom_Tex_000960" OutName="tex_000960" Format="i4" Width="16" Height="16" Offset="0x960" />
-        <Texture Name="object_gi_magicmushroom_Tex_0009E0" OutName="tex_0009E0" Format="i4" Width="32" Height="32" Offset="0x9E0" />
-        <Texture Name="object_gi_magicmushroom_Tex_000BE0" OutName="tex_000BE0" Format="i4" Width="32" Height="32" Offset="0xBE0" />
+    <!-- Get Item Draw: Magic Mushroom -->
+    <File Name="object_gi_magicmushroom" Segment="6"> <!-- "object_gi_mushroom2" -->
```

Comment:
Is this the name of the OoT object?

Reply:
If you mean OoT "Odd Mushroom" which has the same design as MM's "Magic Mushroom", then:

OoT "Odd Mushroom" -> `object_gi_mushroom.xml`
MM "Magic Mushroom" -> `object_gi_magicmushroom.xml`/`object_gi_mushroom2.xml`

Reply:
I'm saying I don't understand what the `<!-- "object_gi_mushroom2" -->` is describing.

Reply:
Ah. z64utils have different names for many of these files, which presumably comes from some og source? every file with a different name in z64utils has a comment next to its file name with that alternative og name